ui/chat/chat.svelte

```diff
@@ -15,6 +16,9 @@
   grooming.onChange(
     state => {
       messages = state.chat;
+      if (state.chat[state.chat.length-1].author !== grooming.username) {
```

Comment:
This can crash if state.chat.length is 0.

Reply:
fixed